demo/sentiment/trainer_config.py

```diff
@@ -29,7 +29,6 @@
     batch_size=128,
     learning_rate=2e-3,
     learning_method=AdamOptimizer(),
-    average_window=0.5,
```

Comment:
This line should be changed into `model_average=ModelAverage(average_window=0.5),`